torch/_inductor/ir.py

```diff
@@ -4315,11 +4309,13 @@ def has_side_effects(self):
             return False
         return get_schema_info(self.op_overload).is_mutable()
 
-    def has_aliasing(self):
+    def get_alias_names(self):
         # TODO - some fallbacks are still OpOverloadPackets
         if not isinstance(self.op_overload, torch._ops.OpOverload):
-            return False
-        return torch._inductor.utils.is_view(self.op_overload)
+            return []
+        if torch._inductor.utils.is_view(self.op_overload):
+            return [inp.get_name() for inp in self.inputs]
```

Comment:
This is a bit weird to me - we're not actually aliasing all the inputs, presumably only one of them.

Moreover, this breaks our invariant that `get_alias_names()` always has length <= 1.

Reply:
We don't have that invariant -- that's only for get_mutation_names